go/hamming: disapprove comments (#1224)
exercism/website-copy: automated-comments/go/hamming/missing_distance_function.md

> The `Distance` function is missing or couldn't be found.

Reviewer: Maybe just 'is missing'? Missing and can't be found are really more or less the same thing, aren't they? Possibly the student forgot to upload the solution file, and just submitted the tests instead (this happens surprisingly often). We might give them a hint if that's the case.

> The `Distance` function is missing or couldn't be found.

Reviewer 2: ```suggestion
Add the function `Distance`, as it is required and currently missing.
```

Perhaps same comment as my suggestion to two-fer

Reviewer 3: Same comment here about potentially converting this to a general comment with parameters.
